sonar-db/src/main/java/org/sonar/db/version/DatabaseVersion.java

```diff
@@ -30,7 +30,7 @@
 
 public class DatabaseVersion {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)